### PR TITLE
[zh] Sync change from English (Issues 25247)

### PR DESCRIPTION
**Pull request by yuandongx**
Sync changes:
- content/zh/docs/reference/kubectl/cheatsheet.md
- content/zh/docs/reference/kubectl/jsonpath.md

Related Issues kubernetes/website#25247

### CODE REVIEW

**Comment by tengqm** [comment]
@yuandongx Please help make sure you use only one commit for a PR. Squash them when you are set, pls.

File: content/zh/docs/reference/kubectl/jsonpath.md [hunk 1]
```diff
@@ -127,17 +127,51 @@ kubectl get pods -o json
...
+# kubectl does not support regular expressions for JSONpath output
+# The following command does not work
+kubectl get pods -o jsonpath='{.items[?(@.metadata.name=~/^test$/)].metadata.name}'
+
+# The following command achieves the desired result
+kubectl get pods -o json | jq -r '.items[] | select(.metadata.name | test("test-")).spec.containers[].image'
+```
+-->
+{{< note >}}
+不支持JSONPath 正则表达式。如需使用正则表达式, 您可以使用如 `jq` 之类的工具。
```
Comment (helight) [inline content/zh/docs/reference/kubectl/jsonpath.md:166]:
```suggestion
不支持 JSONPath 正则表达式。如需使用正则表达式进行匹配操作, 您可以使用如 `jq` 之类的工具。
```

File: content/zh/docs/reference/kubectl/jsonpath.md [hunk 2]
```diff
@@ -127,17 +127,51 @@ kubectl get pods -o json
...
+
+```shell
+# kubectl 的JSONpath输出不支持正则表达式
```
Comment (helight) [inline content/zh/docs/reference/kubectl/jsonpath.md:169]:
```suggestion
# kubectl 的 JSONpath 输出不支持正则表达式
```

File: content/zh/docs/reference/kubectl/jsonpath.md [hunk 3]
```diff
@@ -127,17 +127,51 @@ kubectl get pods -o json
...
+On Windows, you must _double_ quote any JSONPath template that contains spaces (not single quote as shown above for bash). This in turn means that you must use a single quote or escaped double quote around any literals in the template. For example:
+
+```cmd
+kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{'\t'}{.status.startTime}{'\n'}{end}"
+kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{\"\t\"}{.status.startTime}{\"\n\"}{end}"
+```
+{{< /note >}}
 -->
+{{< note >}}
 在 Windows 上，您必须 _double_ 引用任何包含空格的 JSONPath 模板(不是上面 bash 所示的单引号)。反过来，这意味着您必须在模板中的所有文字周围使用单引号或转义的双引号。例如:
```
Comment (helight) [inline content/zh/docs/reference/kubectl/jsonpath.md:145]:
```suggestion
在 Windows 上，您必须用双引号把任何包含空格的 JSONPath 模板(不是上面 bash 所示的单引号)包括起来。反过来，这意味着您必须在模板中的所有文字周围使用单引号或转义的双引号。例如:
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 4]
```diff
@@ -674,14 +706,14 @@ kubectl api-resources --api-group=extensions # "extensions" API 组中的所有
 <!--
 ### Formatting output
 
-To output details to your terminal window in a specific format, you can add either the `-o` or `--output` flags to a supported `kubectl` command.
+To output details to your terminal window in a specific format, add the `-o` (or `--output`) flag to a supported `kubectl` command.
 -->
 ### 格式化输出
 
-要以特定格式将详细信息输出到终端窗口，可以将 `-o` 或 `--output` 参数添加到支持的 `kubectl` 命令。
+要以特定格式将详细信息输出到终端窗口，可以将 `-o`（`--output`）参数添加到支持的 `kubectl` 命令。
```
Comment (helight) [inline content/zh/docs/reference/kubectl/cheatsheet.md:713]:
```suggestion
要以特定格式将详细信息输出到终端窗口，将 `-o`（或者`--output`）参数添加到支持的 `kubectl` 命令中。
```

File: content/zh/docs/reference/kubectl/jsonpath.md (hunk 1 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/jsonpath.md]: Thankes, done.

File: content/zh/docs/reference/kubectl/jsonpath.md (hunk 2 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/jsonpath.md]: Thankes, done.

File: content/zh/docs/reference/kubectl/jsonpath.md (hunk 3 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/jsonpath.md]: Thankes, done.

File: content/zh/docs/reference/kubectl/cheatsheet.md (hunk 4 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/cheatsheet.md]: Thankes, done.

**Comment by yuandongx** [comment]
> @yuandongx Please help make sure you use only one commit for a PR. Squash them when you are set, pls.

Yes, I am the only one commiting to this PR.

**Comment by tengqm** [comment]
@yuandongx You need to squash your commits into one. Read: https://stackoverflow.com/a/5189600

For this one, I'm gonna help using the prow command. Please bear in mind squashing your commits is important for site maintenance.

**Comment by yuandongx** [comment]
> @yuandongx You need to squash your commits into one. Read: https://stackoverflow.com/a/5189600
> 
> For this one, I'm gonna help using the prow command. Please bear in mind squashing your commits is important for site maintenance.

Thanks, I got it.

**Pull request merged**